Solutions/PaloAlto-PAN-OS/Playbooks/PlaybooksOnPrem/PaloAlto-PAN-OS-GetSystemInfo/readme.md

```diff
@@ -0,0 +1,90 @@
+# PaloAlto-PAN-OS-GetSystemInfo OnPrem version
+
+ ## Summary
+
+This playbook allows us to get System Info of a Palo Alto device for a Sentinel alertt. 
+
+When a new Sentinel incident is created, this playbook gets triggered and performs below actions:
+
+1. Gets the various parameters from the alert
+
+2. Gets the System Info for the device in the alert.
+
+3. Creates a Sentinel Incident and updates it with the system info.
+
+
+
+### Prerequisites 
+1. PaloAlto connector needs to be deployed prior to the deployment of this playbook under the same subscription. Relevant instructions can be found in the connector doc page.
+2. Generate an API key.[Refer this link on how to generate the API Key](https://paloaltolactest.trafficmanager.net/restapi-doc/#tag/key-generation)
+3. This playbook only works for Palo Alto incidents. 
+
+
+### Deployment instructions 
+1. Deploy the playbook by clicking on "Deploy to Azure" button. This will take you to deploying an ARM Template wizard.
+
+
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fgithub.com%2Fsocprime%2FAzure-Sentinel%2Fraw%2FPAN-OS-OnPremCustomConnector%2FPlaybooks%2FPaloAlto-PAN-OS%2FPlaybooksOnPrem%2FPaloAlto-PAN-OS-GetSystemInfo%2Fazuredeploy.json)
+[![Deploy to Azure Gov](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fgithub.com%2Fsocprime%2FAzure-Sentinel%2Fraw%2FPAN-OS-OnPremCustomConnector%2FPlaybooks%2FPaloAlto-PAN-OS%2FPlaybooksOnPrem%2FPaloAlto-PAN-OS-GetSystemInfo%2Fazuredeploy.json)
+
+
+2. Fill in the required parameters:
+    * Playbook Name: The playbook name here (e.g. PaloAlto-PAN-OS-GetThreatPCAP)
+    * :  The blob storage account where the threat PCAP will be stored
```

Comment:
This mentioned line is not clear and same parameter (storage account) to fill is not asked in playbook template